src/components/Modal/Modal.stories.tsx

```diff
@@ -59,6 +60,25 @@ export const OpenModalTest = () => {
   );
 };
 
+export const drawHeaderBorderTest = () => {
```

Comment:
Since you have `WithDrawHeaderBorder` above, I'd suggest we remove this `drawHeaderBorderTest` story. They are basically validating the same thing.